admin-auth/src/test/kotlin/com/epam/drill/admin/auth/EnvRepositoryTest.kt

```diff
@@ -50,21 +51,61 @@ class EnvRepositoryTest {
 
     @Test
     fun `given username hash, findById must return the respective user`() = runBlocking {
-        val repository = getRepositoryWithUsers(USER1, USER2)
-        val user = repository.findById("user".hashCode())
+        val repository = prepareEnvUserRepository(
+            user("guest"),
+            user("foobar"),
+            user("admin")
+        )
+
+        val user = repository.findById("foobar".hashCode())
+
+        assertNotNull(user)
+        assertTrue(user.username == "foobar")
+    }
+
+    @Test
+    fun `given lowercase username hash, findById must return the respective user`() = runBlocking {
```

Comment:
`given id generated from username in lowercase, findById must return the respective user`